### ISSUE TITLE
Standard Path Breaks Automation

### ISSUE THREAD

**Issue by Mushroomheadbangers** [houeland/kolproxy#377]
Automation breaks periodically because non-standard items are being used. I've seen it primarily with Smith related items. I also noticed it with the jar of psychosis. 

Some interrupts can be skipped with a "skip" link but others result in a big error page full of red text. This happens everytime when trying to equip Smith related items. 

Doing a hardcore pastamancer run currently.

**Comment by Mushroomheadbangers** [comment]
This issue began on the 1st after the new standard restrictions went into effect.

**Comment by houeland** [comment]
Yeah the most direct problem here is that KoL is still broken and lets you summon the items, but not equip them. /closet-ing the problematic items will fix that.

It's still a problem that the ascension script is tuned for a world where smiths exists, so will take some work to make it run smoothly in the new standard.

**Comment by Mushroomheadbangers** [comment]
Now that the skill can't be used to summon it's not breaking on smith items anymore. I guess that was fixed during tonight's rollover.
